.github/workflows/release-drafter.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - "v0.7"
```

Comment:
Wouldn't this just add the commits from `v0.7` to the same release draft? Or am i missing something?

Reply:
Let try to see if it does! I have heard that it is healthy to live your life on the edge 😆

Reply:
It seems to work for now

Reply:
I checked it, and the 'on:', 'push:' statements are only an indication for GitHub when the workflow should be started. This is ok for our configuration. In the documentation of Release-Drafter it is described that it then only runs on that specific branch (keyword: commitish).